daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -937,8 +956,8 @@ abstract class TestCase(testCaseXML: NodeSeq, val parent: DFDLTestSuite) {
       }
 
       val useSerializedProcessor =
-        if (validationMode == ValidationMode.Full) false
-        else if (defaultValidationMode == ValidationMode.Full) false
+        if (validatorName == "xerces") false
+        else if (defaultValidatorName == "xerces") false
```

Comment:
In theory could someone set the validation="schematron", which shoudl also disable serailized processor? Shoudl we reverse this logic so we use a serialized processor the the validator is not off or limited?

Reply:
Hmm, reversing the logic resulted in failures in our tests since we have to check all the condition before we hit the default true case, but if we switch the logic, we don't end up checking defaultValidationMode...so I left it as is, and updated the validator ticket